code/game/machinery/cryo.dm

```diff
@@ -262,23 +263,26 @@
 	if(href_list["ejectOccupant"])
 		if(!occupant || isslime(usr) || ispAI(usr))
 			return 0 // don't update UIs attached to this object
+		log_game("[key_name(occupant)] ejected by [key_name(usr)] at cryo cell at [COORD(src)]")

```

Comment:
"usr ejected occupant from cryo cell COORD"

(To avoid confusion, put the perpetrator first, then the victim second)